proxy/src/http/id.rs

```diff
@@ -17,7 +17,7 @@ where
     path("ids")
         .and(http::with_shared(registry))
         .and(warp::get())
-        .and(document::param::<registry::Id>(
+        .and(document::param::<String>(
```

Comment:
Would be great if we had `try_param`. :>

Reply:
@xla not sure what you are referring to :) could you elaborate? To avoid misunderstandings it's helpful to read this commit's description in case you haven't yet: https://github.com/radicle-dev/radicle-upstream/pull/681/commits/12529a5b31741cfd52b1f8226ddcb69d62cc080f

Reply:
My comment was in regards that `param` doesn't seem to support types which only implement `TryFrom`, which would be handy in this instance.

Reply:
ah I see! that's right, will follow up on that. As the diff shows, I moved this one back to `String` to have the docs playground working but I will check whether using `try_param::<registry::Id>` solves the issue. :+1:

Reply:
@xla, `document::try_param` is not a thing apparently?

Reply:
That's why I said it would be great if it existed. :dancers:

Reply:
[papoy](https://youtu.be/l0tOJ3gyokk?t=9)! I thought you were proposing to change it :b